gnome-base/gnome-panel/gnome-panel-3.46.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome2 toolchain-funcs
+
+DESCRIPTION="The GNOME Flashback Panel"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-panel/"
+
+LICENSE="GPL-2+ FDL-1.1 LGPL-2.1+"
+SLOT="0"
+IUSE="eds elogind systemd"
+REQUIRED_USE="^^ ( elogind systemd )"
+KEYWORDS="~amd64 ~riscv"
+
+# <libgweather-4.2.0 because of libsoup:3 transition
```

Comment:
I don't think we can do that if we're depending on `sci-geosciences/geocode-glib:2`.

Reply:
My mistake, we depend on >=libgweather-4.2.0 but i forgot to remove the comment.